.github/workflows/tab-finder.yml

```diff
@@ -0,0 +1,39 @@
+# Find tabs in modified text files
+name: Tab finder
+on: [push, pull_request]
+env:
+  MAIN_BRANCH: dev
+jobs:
+  build:
+    name: Tab finder
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # needed to get the full history and pull from upstream
+      - name: Pull from upstream
+        run: |
+          # Update from upstream to avoid including missing commits in the diff
+          git config --global user.name "Nemo" # required on some servers
+          git config pull.rebase false
+          git remote add upstream https://github.com/AliceO2Group/AliceO2.git || exit 1
+          git pull upstream ${{ env.MAIN_BRANCH }} || exit 1
```

Comment:
Would it be easier to just `git fetch ${{ github.event.pull_request.base.sha }}`? Then we don't need to change the `pull.rebase` setting and deal with any other magic that `git pull` might do.

(Also, wouldn't this try to merge the current `dev` into the PR branch? I can't see why we'd want that.)

Reply:
I had `git fetch` in the first version. I cannot remember now why I changed to `git pull`... :-)

Reply:
Probably because of the unrelated files that will appear in the diff if the missing commits are not merged in the current branch.

Reply:
> Probably because of the unrelated files that will appear in the diff if the missing commits are not merged in the current branch.
Yes, that's why I suggested diffing against the merge-base instead of the current dev.